src/Observable.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { Observable } from './Observable'
+
+const impossibru = function (done, msg): () => void {
+    return (): void => {
+        expect(`${msg} WILL NOT HAPPEN`).toBe(''); setTimeout(done)
+    }
+}
+
+describe('Observable', () => {
+    it('can filter messages', done => {
+        const tester = value => {
+            expect(value).toBe(3)
+            setTimeout(done)
```

Comment:
No oki, wzorowałem się na testach z BaseObservable, i wymyśliłem że to jest istotne 👍